napari/view_layers.py

```diff
@@ -321,4 +367,49 @@ def imshow(*args, viewer=None, **kwargs):
         argument is given.
     """
 
-    return _make_viewer_then('add_image', args, kwargs, viewer=viewer)
+    kwargs = dict(
+        viewer=viewer,
+        channel_axis=channel_axis,
+        rgb=rgb,
+        colormap=colormap,
+        contrast_limits=contrast_limits,
+        gamma=gamma,
+        interpolation2d=interpolation2d,
+        interpolation3d=interpolation3d,
+        rendering=rendering,
+        depiction=depiction,
+        iso_threshold=iso_threshold,
+        attenuation=attenuation,
+        name=name,
+        metadata=metadata,
+        scale=scale,
+        translate=translate,
+        rotate=rotate,
+        shear=shear,
+        affine=affine,
+        opacity=opacity,
+        blending=blending,
+        visible=visible,
+        multiscale=multiscale,
+        cache=cache,
+        plane=plane,
+        experimental_clipping_planes=experimental_clipping_planes,
+        title=title,
+        ndisplay=ndisplay,
+        order=order,
+        axis_labels=axis_labels,
+        show=show,
+    )
+
+    args = tuple(
+        [data]
+    )
+
+    viewer, layers = _make_viewer_then(
+        'add_image',
+        # *, # TODO sort out if this needs to be kept/how to handle it
```

Comment:
I'm not sure how to handle this. Can I just remove it? Or can someone make a suggestionn?

Reply:
As far as I understand, the intention was to unify this with the other layers (i.e: no channel splitting stuff). However, I don't think we wanted to release such a big break in VERY public api in a minor release. Correct me if I'm wrong @jni?

If I'm right, then I would say deprecate the use of `channel_axis` (and any other related kwarg) in add_image, and then remove this whole overloaded function in 0.5.0.